docs/machine-learning/ml-datahack-onboarding-questions.md

```diff
@@ -0,0 +1,27 @@
+# Onboarding questions
```

Comment:
These questions could be added to the data exploration so there are not too many files - I would also remove 'hack' part of the file name and the content